README.md

```diff
@@ -121,64 +125,15 @@ __minimum_ios_deployment_target__: str
 __model__: A Core ML model.
 ```
 
-### CLI
-Also you can use command-line script for simplicity:
-```
-convert-onnx-to-coreml [OPTIONS] ONNX_MODEL
-```
-
-The command-line script currently doesn't support all options mentioned above. For more advanced use cases, you have to call the python function directly.
-
 
 ## Installation
 
-### Install From PyPI
-
-```bash
-pip install -U onnx-coreml
-```
-
-### Install From Source
-
-To get the latest version of the converter, install from source by cloning the repository along with its submodules and running the install.sh script. That is,
-
-```bash
-git clone --recursive https://github.com/onnx/onnx-coreml.git
-cd onnx-coreml
-./install.sh
-```
-
-### Install From Source (for contributors)
-
-To get the latest version of the converter, install from source by cloning the repository along with its submodules and running the install-develop.sh script. That is,
-
-```bash
-git clone --recursive https://github.com/onnx/onnx-coreml.git
-cd onnx-coreml
-./install-develop.sh
-```
-
-
-## Running Unit Tests
-
-In order to run unit tests, you need `pytest`.
+To get the latest version of coremltools:
 
 ```shell
-pip install pytest
-pip install pytest-cov
-```
-
-To run all unit tests, navigate to the `tests/` folder and run
-
-```shell
-pytest
-```
-
-To run a specific unit test, for instance the custom layer test, run
-
-```shell
-pytest -s custom_layers_test.py::CustomLayerTest::test_unsupported_ops_provide_functions
+pip install coremltools==4.0b1
 ```
+For the latest changes please see the [release notes](https://github.com/apple/coremltools/releases/).
```

Comment:
Again, please keep the instructions to document coremltools in that documentation. This will get stale quickly.

Reply:
Removed.